src/test/scala-2.11/markets/participants/LiquiditySupplierSpec.scala

```diff
@@ -0,0 +1,64 @@
+package markets.participants
+
+import akka.actor.ActorSystem
+import akka.agent.Agent
+import akka.testkit.{TestProbe, TestActorRef, TestKit}
+
+import markets.orders.limit.LimitOrderLike
+import markets.tickers.Tick
+import markets.tradables.TestTradable
+import org.scalatest.{Matchers, GivenWhenThen, FeatureSpecLike}
+
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1653013742/issues/source?bid=2921361&fileBranchId=3037089#l13)